commit cb0732f60c20e8f70b641a7b319ed3015d3950ac
Author: Matt Robenolt <matt@ydekproductions.com>
Date:   Wed May 30 14:47:29 2018 -0700

    fix(tests): Don't assume local Postgres is accessible over Unix socket
    
    This should be no change in behavior for anyone that does use a unix
    socket, since it should also be bound to 127.0.0.1. But for those of us
    that do not have a unix socket, it should make tests actually work.

diff --git a/src/sentry/utils/pytest/sentry.py b/src/sentry/utils/pytest/sentry.py
index 90fad556b3..8abdbb82d6 100644
--- a/src/sentry/utils/pytest/sentry.py
+++ b/src/sentry/utils/pytest/sentry.py
@@ -51,6 +51,7 @@ def pytest_configure(config):
                     'ENGINE': 'sentry.db.postgres',
                     'USER': 'postgres',
                     'NAME': 'sentry',
+                    'HOST': '127.0.0.1',
                 }
             )
             # postgres requires running full migration all the time
